Tribler/community/multichain/database.py

```diff
@@ -232,7 +230,7 @@ def contains(self, hash_requester):
         :param hash_requester: The hash_requester that is queried
         :return: True if the block exists, else false.
         """
-        db_query = u"SELECT hash_requester FROM multi_chain WHERE hash_requester == ? LIMIT 1"
+        db_query = u"SELECT hash_requester FROM multi_chain WHERE hash_requester = ? LIMIT 1"
         db_result = self.execute(db_query, (buffer(hash_requester),)).fetchone()
```

Comment:
The python-sqlite module requires that all BLOB type parameters are presented as python type buffer().